spec/tasks/fastly_spec.rb

```diff
@@ -6,14 +6,16 @@
     PracticalDeveloper::Application.load_tasks
   end
 
-  describe "#update_whitelisted_params" do
+  describe "#update_safe_params" do
     it "doesn't run if Fastly isn't configured" do
-      allow(ApplicationConfig).to receive(:[]).with("FASTLY_API_KEY").and_return(nil)
-      allow(ApplicationConfig).to receive(:[]).with("FASTLY_SERVICE_ID").and_return(nil)
-      allow(ApplicationConfig).to receive(:[]).with("FASTLY_WHITELIST_PARAMS_SNIPPET_NAME").and_return(nil)
-      allow(FastlyVCL::WhitelistedParams).to receive(:update)
-      Rake::Task["fastly:update_whitelisted_params"].invoke
-      expect(FastlyVCL::WhitelistedParams).not_to have_received(:update)
+      %w[FASTLY_API_KEY FASTLY_SERVICE_ID FASTLY_SAFE_PARAMS_SNIPPET_NAME].each do |var|
+        allow(ApplicationConfig).to receive(:[]).with(var).and_return(nil)
+      end
+      allow(FastlyVCL::SafeParams).to receive(:update)
+
+      Rake::Task["fastly:update_safe_params"].invoke
```

Comment:
We need to fix this or comment it out

https://github.com/thepracticaldev/dev.to/pull/7486

Reply:
Done! See https://github.com/thepracticaldev/dev.to/pull/7459/commits/da890c7d91c3ce62b8f4f2727bb9a3754a76c652